benchmarks/build.gradle

```diff
@@ -1,6 +1,8 @@
+apply plugin: 'me.champeau.jmh'
```

Comment:
This plugin has a number of problems. It is trivial to set this up ourselves - I can provide some examples where we've done this internally.